alpha/veneer/basic/README.md

```diff
@@ -0,0 +1,117 @@
+### Basic Veneer
+To simplify operation with File-Based Catalogs (FBCs), we can first offer the option to eliminate information which can be retrieved from existing registries when we generate the FBC. The `basic veneer` is an input schema which is effectively a subset of an FBC with support to do just this. Users of the `basic veneer` can provide all existing elements of an FBC, but short-hand the `olm-bundle` objects, requiring only the bundle image reference and greatly reducing the size/complexity of the resulting document.
```

Comment:
```suggestion
The `basic veneer` is an input schema to simplify working with File-Based Catalogs (FBCs). It acts like a subset of an FBC with support to do just this. The `basic veneer` reduces the size and complexity of the resulting FBC removing redundant information which can be retrieved from existing registries.   Users add the existing elements of an FBC along with the bundle image reference to shorthand the required `olm-bundle` objects. 
```
Maybe this changes the intended meaning too much. Please feel free to disregard if it isn't helpful.

Reply:
You're absolutely right that the verbiage there needs de-tangling and simplification.  
I took another swing at it with that intent.  PTAL and let's see if we can converge.

Reply:
Awesome! LGTM!